.travis.yml

```diff
@@ -118,7 +118,7 @@ install:
       export PATH=${DEPS_DIR}/cmake/bin:${PATH}
 
     elif [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
-      brew install cmake boost libzip libstxxl libxml2 lua51 luabind tbb GDAL
+      brew install cmake boost libzip libstxxl libxml2 lua51 luabind tbb GDAL md5sha1sum
```

Comment:
`libzip`? Is this supposed to install `zlib` since `libzip` != `libz`?

Also, do we build the tools on Travis? If not drop the `GDAL` dependency.